tests/test_sanity_sample.py

```diff
@@ -682,11 +689,14 @@ def icnet_desc(x: TestCaseDescriptor):
     inception_v3_desc(AutoQDescriptor()).batch(2),
     inception_v3_desc(AutoQDescriptor()).staged(),
     resnet18_desc(AutoQDescriptor()).batch(2),
+    resnet18_desc(AutoQDescriptor()).dump_debug(True),
```

Comment:
how much time does it take? I am afraid, these tests are too heavy for checking debug capabilities. Any chance to localize it in something lightweight (e.g. in test_autoq_precision_init) or in an unit test just for quantization env?

Reply:
@ljaljushkin , it is a legitimate concern, I checked on the jenkins build on this patch and compared it to previous jenkins build, the 3 new tests have incurred 6 additional minutes. 

Unfortunately, the dump setting is used throughout the AutoQ, not for a particular unit, it resorts to sanity sample test kind of pipeline. Input for this test is also from json config. here is my suggestion, we can add this flag to the existing tests, to keep the test time impact minimal while coverage when this corner is broken. Please let me know if this works, I will push the changes.

Reply:
I will try to come up with more lightweight test

Reply:
@vuiseng9, let's add this config to the existing tests